Sources/AlgoliaSearchClient/Helpers/Coding/Wrappers/BoolContainer.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  BoolContainer.swift
+//  
+//
+//  Created by Vladislav Fitc on 17/07/2020.
+//
+
+import Foundation
+
+struct BoolContainer: RawRepresentable, Codable {
```

Comment:
Can we write a small comment on the purpose of the class, for future readers to understand

Reply:
Done ✅  Add as well the explanation for `StringNumberContainer` introduced for almost the same purpose.